api/proto/teleport/recordingencryption/v1/recording_encryption.proto

```diff
@@ -51,3 +66,22 @@ message RecordingEncryption {
   RecordingEncryptionSpec spec = 5;
   RecordingEncryptionStatus status = 6;
 }
+
+// A rotated keypair with its fingerprint.
+message RotatedKeySpec {
+  string fingerprint = 1;
+  types.EncryptionKeyPair encryption_key_pair = 2;
+}
+
+// The empty status of a RotatedKey.
+message RotatedKeyStatus {}
+
+// A previously rotated encryption key for session recordings kept for future replay.
+message RotatedKey {
```

Comment:
I'm curious about the decision to add `RotatedKey` as a separate resource rather than a subkind or even just a status within `RecordingEncryption`, was this discussed in the RFD? The only different right now seems to be the addition of `fingerprint` and the different statuses, which could be united.

Reply:
Putting it in the status was discussed in the RFD, but we opted for a separate resource to avoid running into size limits in the case that keys are rotated often or kept for a long time. I never considered making it a subkind (mostly because I didn't realize what that was 😅 ), but it does seem like that could also work. The main difference being that there should only ever be one `RecordingEncryption` but you can have any number of `RotatedKey`s. That doesn't mean they can't share the same schema though.

Would there be benefits to using a subkind other than reducing the number of unique resources?